source/js/embed-tf.js

```diff
@@ -0,0 +1,29 @@
+let EmbedTF = {
+  init: function() {
+    document.addEventListener("DOMContentLoaded", () => EmbedTF.setup());
+  },
+  setup: function() {
+    let formMeta = document.querySelector("meta[name=typeform-formurl]");
+
+    if (!formMeta) {
+      return;
+    }
+
+    let popup = window.typeformEmbed.makePopup(
```

Comment:
We definitely want to put this behind `if (window.typeformEmbed) { ... }` because that property may not exist (bad networking, script blockers, etc)